ganga/GangaCore/Core/GangaRepository/container_controllers.py

```diff
@@ -255,6 +281,15 @@ def singularity_handler(database_config, action, gangadir):
     if action not in ["start", "quit"]:
         raise NotImplementedError(f"Illegal Opertion on container")
 
+    has_correct_hash = check_sif_file_hash(sif_file)
+    if not has_correct_hash:
+        import sys
+        logger.fatal((
+            "sif_file does not match the expected hash." +
+            " Delete it to automatically download the correct file."
```

Comment:
Tell the location of the faulty file in this message. Otherwise the user will hunt around to delete a file with no idea about where it is.

Reply:
This looks resolved to me, so I'll merge and have a test.